src/Common/src/Interop/Windows/user32/Interop.MessageBox.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
+
+internal partial class Interop
+{
+    internal partial class User32
+    {
+        internal const int MB_ICONHAND = 0x00000010;
```

Comment:
No, there are tons more which determine the layout of the message box.  So I'll leave what we have for now.